### PR TITLE
Some modifications

### PR DESCRIPTION
**Pull request by Maitri-Shah24**
Modified all the previous comment given by Ayan bhai, Nisarg and Shreya.

### CODE REVIEW

**Comment by Maitri-Shah24** [comment]
Please try to review full code and not only loaded differences

File: src/main/java/com/example/Board.java [hunk 1]
```diff
@@ -100,7 +92,6 @@ else if (isWinningLine(getColumnPositions(i), cpuPositions))
                 return "Computer wins! Sorry (\u2060 \u2060⚈̥\u2060⌢\u2060⚈̥\u2060)" + playerName;
```
Comment (shreya6323) [inline src/main/java/com/example/Board.java]: Avoid repeating the print statement related to winner

File: src/main/java/com/example/Board.java
Comment (shreya6323) [inline src/main/java/com/example/Board.java:1]: in reset() declare 2 as constant

File: src/main/java/com/example/Game.java [hunk 2]
```diff
@@ -11,7 +11,7 @@ class Game {
 
     public Game() {
         System.out.println("Bored!Let's play a Game!!\uD83C\uDFB2");
-        System.out.println("Enter the grid size: (3 for 3x3, 4 for 4x4)");
+        System.out.println("Enter the grid size: (3 for 3x3, 4 for 4x4 etc)");
```
Comment (shreya6323) [inline src/main/java/com/example/Game.java:14]: Avoid using print statements in Constructor

File: src/main/java/com/example/Game.java
Comment (shreya6323) [inline src/main/java/com/example/Game.java:1]: Try to modularize the things in constructor. It is preferred to have only initialization things over there

Comment (shreya6323) [inline src/main/java/com/example/Game.java:1]: According to Tic-Tac-Toe ruleset, the one with 'X' will get the first chance, so try to incorporate that logic in the code.

**Comment by shreya6323** [comment]
> Please try to review full code and not only loaded differences
Also check the comment I have mentioned in your previous commit of this branch

File: src/main/java/com/example/HumanPlayer.java
Comment (NisargPipaliya) [inline src/main/java/com/example/HumanPlayer.java:1]: Update the print statements according to board size.

Comment (NisargPipaliya) [inline src/main/java/com/example/HumanPlayer.java:1]: The way of checking vaild input is not that efficient, i.e. need to create exception object, throw it, catch it, print message.
Instead assume user provides invalid input, and validate it before using it.

**Review by NisargPipaliya — CHANGES_REQUESTED** [review]

**Comment by Maitri-Shah24** [comment]
> > Please try to review full code and not only loaded differences
> > Also check the comment I have mentioned in your previous commit of this branch.
Done

File: src/main/java/com/example/HumanPlayer.java
Comment (Maitri-Shah24) [reply src/main/java/com/example/HumanPlayer.java:1]: Done

File: src/main/java/com/example/Game.java (hunk 2 above)
Comment (Maitri-Shah24) [reply src/main/java/com/example/Game.java:14]: This Game constructor rn is used to create instance in TicTacToe class if I don't print statements here then would have to call all the methods in TicTacToe class with instance of Game class which would make a mess. Would take care next time.

File: src/main/java/com/example/Board.java
Comment (Maitri-Shah24) [reply src/main/java/com/example/Board.java:1]: That 2 is never going to change as it is the denoting separators, intersection symbols for printing board. So I think there is no use to declare it as a constant bcz the method for printing board is not going to change.

File: src/main/java/com/example/Board.java (hunk 1 above)
Comment (Maitri-Shah24) [reply src/main/java/com/example/Board.java]: Done

**Review by NisargPipaliya — APPROVED** [review]

**Review by shreya6323 — APPROVED** [review]
Approved !

**Pull request merged**